projects/step-frontend/src/lib/modules/execution/components/execution-progress/execution-progress.component.html

```diff
@@ -14,6 +14,31 @@
 @if (execution) {
   <section [class.small-screen]="_isSmallScreen$ | async">
     <ng-template #executionTabs>
+      @if (
+        autoscalingStatus?.statusDescription ||
+        autoscalingStatus?.tokenCountStarted ||
+        autoscalingStatus?.tokenCountStarted
+      ) {
+        <step-alert [type]="AlertType.INFO" class="auto-scaling-status">
+          <div>Auto scaling status</div>
+          <div>{{ autoscalingStatus?.statusDescription }}</div>
+          @if (autoscalingStatus?.tokenCountStarted) {
+            <div>Token count started</div>
```

Comment:
Would prefer to display something like: "Progress: x/y agent token started" where x = tokenCountStarted and y = tokenCountTarget